pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: news_post
+title: "2017 Fukuoka Ruby Award Competition - Inscrições serão avaliadas por Matz"
+author: "Fukuoka Ruby"
+translator: jcserracampos
+date: 2016-10-20 00:00:00 +0000
+lang: pt
+---
+
+Prezados Entusiastas Ruby,
+
+O Governo de Fukuoka, Japão, juntamente com "Matz" Matsumoto gostaria de lhe
+convidar para se inscrever na seguinte competição Ruby. Se você já desenvolveu um
+programa Ruby interessante, por favor sinta-se encorajado a participar.
+
+2017 Fukuoka Ruby Award Competition - Grande Prêmio - 1 Milhão de Yens!
+
+Prazo de inscrição: 27 de dezembro de 2016
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de painelista selecionarão os vencedores da Fukuoka Competition.
+O grande prêmio para a Fukuoka Competition são um milhão de yens.
+Ganhadores do grande prêmio anteriores incluem Rhomobile (EUA) e APEC Climate Center(Coreia)
+
+[http://myfukuoka.com/category/news/ruby-news/](http://myfukuoka.com/category/news/ruby-news/)
+
+Programas inscritos na competição não precisam ser escritos inteiramente em Ruby
+mas devem aproveitar-se de uma das características únicas do Ruby.
+
+Projetos devem ter sido desenvolvidos ou concluídos dentro dos 12 meses passados
+para serem elegíveis. Por favor, visite os seguintes sites da Fukuoka para informações
+adicionais ou para se inscrever
+
+[http://www.digitalfukuoka.jp/events/114](http://www.digitalfukuoka.jp/events/114)
+ou
+[http://myfukuoka.com/events/2017-fukuoka-ruby-award-guidelines-for-applicants/](http://myfukuoka.com/events/2017-fukuoka-ruby-award-guidelines-for-applicants/)
+
+[http://www.digitalfukuoka.jp/uploads/event_detail/file/305/RubyAward_ApplicationForm_2017.doc](http://www.digitalfukuoka.jp/uploads/event_detail/file/305/RubyAward_ApplicationForm_2017.doc)
+
+Por favor, envie o formulário de inscrição para award@f-ruby.com.
+
+Neste ano, nós temos os seguintes prêmios especiais:
+
+O vencedor do AWS Prize receberá:
+
+* Amazon Fire Tablet (sujeito a alteração)
+* Consultoria técnica de arquiteto da AWS
+
+O vencedor do GMO Pepavo Prize receberá:
+
+* Uma cesta de presentes cheia de comidas locais e petiscos (no valor de 30.000 yens)
+* Um certificado de 50.000 yens para serviços de domínio
+
+O vencedor do IIJ GIO Prize receberá:
+* Um cupom gratuíto no valor de 500.000 yens do IIJ GIO (até 6 meses)
+
+O vencedor do Salesforce Prize receberá:
+
+* Novidades da salesforce.com
+
+"Matz estará testando e revisando completamente seu código fonte, então
```

Comment:
`irá testar e revisar completamente..` I think it sounds better without the "gerundismo"